integration/ibc/cmd_ibc_test.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 func TestCreateModuleWithIBC(t *testing.T) {
+	t.Skip("skipping test as IBC isn't available with v0.52 yet") // https://github.com/ignite/cli/pull/4289
```

Comment:
Can we create an issue so we don't lose track?

Reply:
https://github.com/ignite/cli/issues/4387